tsconfig.json

```diff
@@ -13,9 +13,13 @@
     "sourceMap": true,
     "rootDir": "."
   },
+  "include": [
```

Comment:
I have deliberately left both sections as they are now shared by the `Typescript` compiler and the `tslint`. I haven't tested removing the `exclude` section.